setup.py

```diff
@@ -1,17 +1,17 @@
 try:
-    from setuptools import setup,
+    from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
 config = {
-    name='pynio',
-    version='0.1dev',
-    packages = ['pynio'],
-    install_requires = [
+    'name': 'pynio',
+    'version': '0.1dev',
```

Comment:
I'm not too concerned about what the version is called. I saw 'dev' being used somewhere and thought it was a good idea. Given that we're probably not going to be strict about updating versions any time soon, I figured it was a good way to imply that things are changing rapidly.